### PR TITLE
Support `SnapService` to manage all services within the same snap.

### PR DESCRIPTION
**Pull request by chanchiwai-ray**
Remove `service_name` attribute and argument from `SnapService`, since snap's `start`  and `stop` operation supports all services if the service name is not provided. Also, update the `SnapService.is_active()` to accept a service name args, this will allow for checking the status of a particular service within the snap.

Closes: #10

### CODE REVIEW

**Review by jneo8 — COMMENTED** [review]
Wouldn't it be weird that you need to start or stop every service?

**Comment by chanchiwai-ray** [comment]
> Wouldn't it be weird that you need to start or stop every service?

Normally, the services within the snap should work together, so start or stop all services isn't that bad. Also, in our case, it does not matter.

**Review by jneo8 — APPROVED** [review]

**Comment by chanchiwai-ray** [comment]
Rebased onto main

File: src/service.py [hunk 1]
```diff
@@ -9,24 +9,23 @@
...
+    def __init__(self, client: snap.Snap) -> None:
         """Initialize the class."""
         self.snap_client = client
-        self.snap_service = snap_service
 
-    def is_active(self) -> bool:
+    def is_active(self, service: str) -> bool:
         """Return True if the snap service is active."""
-        return self.snap_client.services.get(self.snap_service, {}).get("active", False)
+        return self.snap_client.services.get(service, {}).get("active", False)
```
[the remark this answers is not in the document]
Comment (chanchiwai-ray) [reply src/service.py:20]: There's not direct API for this, but we can implement it ourselves. I suppose you want 

```python
class SnapService:
    def is_active(self, services: Optional[list[str]]):
        ...

# usage
sc = SnapService(...)
sc.is_active(["a", "b", "c"]) # return True if all "a", "b", "c" are active
sc.is_active() # returnTrue if all services are active
```
?

[the remark this answers is not in the document]
Comment (chanchiwai-ray) [reply src/service.py]: Okay, that's more consistent with the rest of methods. Done

**Review by jneo8 — DISMISSED** [review]

**Comment by chanchiwai-ray** [comment]
tested again, the new `is_active` method works as expected

**Review by jneo8 — APPROVED** [review]

**Comment by chanchiwai-ray** [comment]
rebased

**Review by jneo8 — APPROVED** [review]

**Pull request merged**